ClassNotFoundException: org.ejml.simple.ConstMatrix in dev-align-merge branch of coatjava (CVT alignment)

to reproduce:
/work/clas12/users/gotra/test/devalignm/coatjava/coatjava/bin/recon-util -y /work/clas12/users/gotra/test/devalignm/alignment/clas12/scripts/generic/align_zf.yaml -i /work/clas12/users/gotra/test/devalignm/alignment/clas12/scripts/generic/19323r.hipo -o test.hipo
Exception:
[CONFIGURE][CVTEngine] ---->  Setting timestamp :
Exception in thread "main" java.lang.NoClassDefFoundError: org/ejml/simple/ConstMatrix
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:421)
	at java.base/java.lang.Class.forName(Class.java:412)
	at org.jlab.clas.reco.EngineProcessor.addEngine(EngineProcessor.java:214)
	at org.jlab.clas.reco.EngineProcessor.main(EngineProcessor.java:393)
Caused by: java.lang.ClassNotFoundException: org.ejml.simple.ConstMatrix
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 5 more

--- a/common-tools/clas-tracking/pom.xml
+++ b/common-tools/clas-tracking/pom.xml
@@ -16,8 +16,8 @@
   <dependencies>
     <dependency>
         <groupId>org.ejml</groupId>
-        <artifactId>ejml-all</artifactId>
-        <version>0.44.0</version>
+        <artifactId>ejml-simple</artifactId>
+        <version>0.40</version>
     </dependency>
     <dependency>
       <groupId>org.jlab.clas</groupId>

